servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/tests/AbstractRelativeReferences.java

```diff
@@ -87,13 +89,16 @@ void createFixtures() {
          etl1          c3
                          \
          etl2             c4 -- c5
+                                 ^
+                               tag1
     */
     assumeTrue(outer.isNewModel());
     Branch base = outer.createBranchV2("base");
     base = outer.commitV2(base, key1, table1);
     c1 = base.getHash();
     base = outer.commitV2(base, key2, table2);
     c2 = base.getHash();
+    outer.createTagV2("tag0", base);
```

Comment:
Looks like `tag0` is missing in the diagram above (`tag1`'s added).

Reply:
Actually we won't need `tag0` anymore, I forgot to remove it.